qermit/zero_noise_extrapolation/zne.py

```diff
@@ -190,11 +190,11 @@ def odd_gate(circ: Circuit, noise_scaling: float) -> Circuit:
 
         for command in c_dict["commands"]:
 
+            command_circ_dict= {key: val for key, val in c_dict.items() if key != "commands"}
```

Comment:
this can be put outside the loop, those values don't change between iterations

Reply:
ah yeah, cheers!